frontend/src/components/ProfileHeader.js

```diff
@@ -0,0 +1,10 @@
+import React from 'react'
+
+// STORES PFP IMAGE + BIO FOR REUSABILITY 
+function ProfileHeader() {
+  return (
+    <div>ProfileHeader</div>
+  )
+}
+
+export default ProfileHeader
```

Comment:
This isn't being used anywhere, it would be considered dead code. Best practice is to not check it in until it's used.